en_US.json

```diff
@@ -244,6 +244,7 @@
 	"certificate.deploy.inprogress.modal.title": "Deploying Certificate",
 	"certificate.deploy.success.modal.message": "The certificate has been successfully deployed.",
 	"certificate.deploy.success.modal.title": "Certificate Deployed",
+	"certificate.dns01challenge.confirm.cancel.verifying": "Do you want to discard verifying DNS-01 challenge?",
```

Comment:
I suggest it be rephrased as 
"Do you want to discard verification of the DNS-01 challenge?"

Reply:
Thanks for your suggestion.
I fixed it.